analyzer/codechecker_analyzer/cmd/checkers.py

```diff
@@ -65,8 +63,12 @@ def get_warnings(env=None):
 
     command = [diagtool_bin, 'tree']
     try:
-        result = subprocess.check_output(command, env=env,
-                                         universal_newlines=True)
+        result = subprocess.check_output(
```

Comment:
Since we always call this function with 
```
           universal_newlines=True,
            encoding="utf-8",
            errors="ignore")
```

as the last arguments, maybe it would be worth to add a lightweight wrapper with those arguments and use that to make call sites more concise.

Reply:
And same for other subprocess calls.

Reply:
There were some solutions where we made wrapper function calls around Popen earlier but over the time the argument list needed to extended more and more until we almost got the function signature of Popen. It might save some arguments but I would not go down on that road again.

Reply:
Since we have quite a few subprocess calls the main advantage would not only be the shorter call sites but making it harder to forget to set the encoding. The same could be done for `open` calls, so this would not be only an aesthetic change but it would make it harder to commit certain kinds of bugs. But I think it would be ok to do this refactoring as a separate pull request.